pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -24,6 +24,54 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// SanitizeDataSourceZero uses /dev/zero as sanitize source
+	SanitizeDataSourceZero SanitizeDataSourceProperty = "zero"
```

Comment:
I would expect these consts to be defined in the same package as sanitize.go. Nothing else should need to reference them.

Reply:
I moved them to the cleanup.go in the same dir, let me know if this is not acceptable.